nnvm/python/nnvm/frontend/onnx.py

```diff
@@ -824,6 +824,20 @@ def from_onnx(self, graph, opset):
             else:
                 self._num_input += 1
                 self._nodes[i_name] = _sym.Variable(name=i_name)
+        # get list of unsupported ops
+        convert_map = _get_convert_map(opset)
+        unsupported_ops = set()
+        for node in graph.node:
+            op_name = node.op_type
+            if op_name not in convert_map and op_name != 'Constant':
```

Comment:
should we check the _identity_list?  even though currently it is empty but it would be good to make logic complete.